pkg/operator/etcdendpointscontroller/etcdendpointscontroller_test.go

```diff
@@ -50,6 +52,9 @@ func TestBootstrapAnnotationRemoval(t *testing.T) {
 			name: "NewConfigMapAfterDeletion",
 			objects: []runtime.Object{
 				u.BootstrapConfigMap(u.WithBootstrapStatus("complete")),
+				u.FakeNode(fmt.Sprintf("%016x", etcdMembers[0].ID), u.WithMasterLabel(), u.WithNodeInternalIP(dnshelpers.GetIPFromAddressWithPanic(etcdMembers[0].PeerURLs[0]))),
```

Comment:
@hasbro17  @dusk125  I am using the same way the `memberID` is encoded on the configMap within the test. Yet it never get the same value and the tests fails !!